app/controllers/api/v0/highlights_swagger.rb

```diff
@@ -278,12 +278,16 @@ class Api::V0::HighlightsSwagger
         end
       end
       parameter do
-        key :name, :color
+        key :name, :colors
         key :in, :query
-        key :type, :string
+        key :type, :array
+        key :collectionFormat, :csv
         key :required, false
-        key :enum, VALID_HIGHLIGHT_COLORS
-        key :description, 'Limits results to this highlight color.'
+        key :description, 'Limits results to these highlight colors.'
+        items do
+          key :type, :string
+          key :enum, VALID_HIGHLIGHT_COLORS
```

Comment:
our version of swagger codegen does not generate validations for enums in an array, which is why I had to write them separately